hoodie-common/src/test/resources/simple-test-evolved.avro

```diff
@@ -0,0 +1,12 @@
+{
+"namespace": "example.avro",
+ "type": "record",
+ "name": "User",
+ "fields": [
+     {"name": "field1", "type": ["null", "string"], "default": null},
+     {"name": "field2", "type": ["null", "string"], "default": null},
+     {"name": "name", "type": ["null", "string"], "default": null},
+     {"name": "favorite_number",  "type": ["null", "long"], "default": null},
+     {"name": "favorite_color", "type": ["null", "string"], "default": null}
+ ]
+}
```

Comment:
Since this avro schema is used interchangeably across SchemaTestUtils and InputFormatTestUtil.